app/os_macos.go

```diff
@@ -563,6 +574,16 @@ func gio_onFocus(view C.CFTypeRef, focus C.int) {
 	w.SetCursor(w.cursor)
 }
 
+//export gio_onResponder
+func gio_onResponder(view C.CFTypeRef, first C.int) {
+	// That function is called when some child view becomes first responder.
+	w := mustView(view)
+	w.focused = first == 1
+	if w.w.d != nil {
```

Comment:
We don't check `w.w.d` for nil elsewhere, why here? If it's really necessary, it smells like a different problem, perhaps that `SetDriver` is called too late.

Reply:
Without that check it crashes. I'm not sure exactly why. I also tried to wrapper it into `w.run()` (the main thread stuff), and still crashing.

Reply:
What's the crash dump? Maybe we can figure out why.

Reply:
I need to check again, but that is something like: "sending events before drivers is ready".

Reply:
I thought so, which is why I commented "perhaps the SetDriver is called too late". You can try moving that call earlier, or you may have to construct a window in a two-step process: first create the window, then make it visible/key/whatever.